<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>推荐抛售股票</string>
  </property>
  <widget class="QTableWidget" name="SellTable">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>521</width>
     <height>401</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>股票编号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>股票单价</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>抛售数量</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>抛售单价</string>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>440</y>
     <width>511</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="btn_sell">
      <property name="text">
       <string>确认抛售</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
